internal/knowledge/querier.go

```diff
@@ -72,7 +73,12 @@ func (q *Querier) queryInternal(ctx context.Context, cypherQuery string) (*Queri
 		return nil, translation.Query, err
 	}
 
-	logrus.Debugf("Found results in %dms", s.Execution/time.Millisecond)
+	executionTime := s.Execution.Seconds() * 1000
```

Comment:
why not simply use .Milliseconds() instead?

Reply:
https://pkg.go.dev/time#Duration.Milliseconds

Reply:
To avoid casting to float afterwards when using .Observe() , but it makes sense to use this instead.